docs/source/datasets/index.rst

```diff
@@ -6,10 +6,10 @@ Datasets
 
 Here we cover the realistic simulated datasets, which are analogous to "real world" administrative records such as tax documents
 and routinely generated files of social security numbers, that users can generate using Pseudopeople for developing and testing Entity
```

Comment:
```suggestion
and routinely generated files of social security numbers, that users can generate using pseudopeople for developing and testing Entity
```
I just realized the capitalization for 'pseudopeople' is inconsistent with the rest of documentation - it should all be lower-case right?

Reply:
Yes, thanks, pseudopeople should be lower case. I think there are several minor edits like this that we should do to improve the documentation, but I haven't been prioritizing them since we're still working on getting the main content in line with what the engineers are doing. For the sake of limiting the scope of each PR, I think I'm not going to make this change in this PR, but I'll create a JIRA ticket to take a pass at editing the Datasets page. Note that there are a few more edits to make on this page, e.g., at least one more instance of capitalized "Pseudopeople", and "Entity Resolution" should also be lower case.